backend/worker/worker.go

```diff
@@ -482,6 +482,31 @@ func (w *Worker) PublicWorker(ctx context.Context) {
 		wg.Wait()
 		_wg.Done()
 	}(&wg)
+	go func(_wg *sync.WaitGroup) {
+		wg := sync.WaitGroup{}
+		wg.Add(parallelBatchWorkers)
+		buffer := &KafkaBatchBuffer{
+			messageQueue: make(chan *kafkaqueue.Message, DefaultBatchFlushSize*(parallelBatchWorkers+1)),
+		}
+		for i := 0; i < parallelBatchWorkers; i++ {
+			go func(workerId int) {
+				s, wCtx := tracer.StartSpanFromContext(ctx, "kafkaWorker", tracer.ResourceName("worker.kafka.batched.outer.traces"))
+				defer s.Finish()
+				k := KafkaBatchWorker{
+					KafkaQueue:          kafkaqueue.New(wCtx, kafkaqueue.GetTopic(kafkaqueue.GetTopicOptions{Type: kafkaqueue.TopicTypeBatched}), kafkaqueue.Consumer, nil),
```

Comment:
Talked w/ @Vadman97 about creating a new topic for this rather than using the same one as logs. Going to look at setting this up to avoid congestion on `TopicTypeBatched`.